tensorflow/tools/quantization/quantize_graph.py

```diff
@@ -453,7 +453,7 @@ def rewrite(self, output_node_names):
 
   def round_nodes_recursively(self, current_node):
     """The entry point for simple rounding quantization."""
-    if (current_node.name in self.already_visited) and self.already_visited[current_node.name]:
+    if current_node.name in self.already_visited:
```

Comment:
This was actually changed recently in #9393, but this actually brings the column width past 80 characters. This syntax is used in the rest of the file, and since the values in `self.already_visited` are only ever set to `True`, we can stick to this more concise version.